frontend/src/util/apiWrapper.js

```diff
@@ -2,9 +2,10 @@ import { Semaphore } from 'await-semaphore';
 import axios from 'axios';
 import { createDefaultBundleName, pathIsArchive, getArchiveExt } from './worksheet_utils';
 
-export const get = (url, params) => {
+export const get = (url, params, options) => {
```

Comment:
```suggestion
export const get = (url, params, options={}) => {
```

Reply:
@epicfaace is this needed? the spread operator is capable of handling undefined i think

Reply:
Not needed, but it would make it clearer that this argument is optional if someone reads the function definition.